modules/optimization/doc/content/modules/optimization/theory/InvOptTheory.md

```diff
@@ -0,0 +1,373 @@
+
+The alternative that we will use is the adjoint method which scales independently to the number of optimization parameters.   The adjoint method requires one additional solve of the same complexity as the original forward problem.  The adjoint equation given in [eq:adjoint] is found by setting the adjoint variable, $\lambda$, equal to the first two terms of [eq:objectiveGrad2] and then rearranging terms to give
+\begin{equation} \label{eq:AdjEqn}
+	\begin{aligned}
+	&\lambda^\top= \frac{\partial f}{\partial \mathbf{u}}\left(\frac{\partial \mathcal{R}}{\partial \mathbf{u}}\right)^{-1}  \\
+	&\rightarrow \lambda^\top \left(\frac{\partial \mathcal{R}}{\partial \mathbf{u}}\right)= -\frac{\partial f}{\partial \mathbf{u}}\left[\left(\frac{\partial \mathcal{R}}{\partial \mathbf{u}}\right)^{-1} \left(\frac{\partial \mathcal{R}}{\partial \mathbf{u}}\right)\right] \\
+	&\rightarrow \lambda^\top \left(\frac{\partial \mathcal{R}}{\partial \mathbf{u}}\right)= -\frac{\partial f}{\partial \mathbf{u}} \\
+	&\rightarrow \left[\lambda^\top \left(\frac{\partial \mathcal{R}}{\partial \mathbf{u}}\right)\right]^\top= -\left[\frac{\partial f}{\partial \mathbf{u}}\right]^\top \\
+	&\rightarrow \left(\frac{\partial \mathcal{R}}{\partial \mathbf{u}}\right)^\top \lambda = -\left[\frac{\partial f}{\partial \mathbf{u}}\right]^\top \\
+\end{aligned}
+\end{equation}
+where each step of the derivation has been included as a reminder of how [eq:adjoint] is obtained.  The next [section](#sec:adjoint) uses an alternative approach to determine the adjoint equation based on Lagrange multipliers [!citet](walsh2013source).
+
+[comment1]: <> (% ----------------------------------------------------------------------------------------------------------%)
+
+## Adjoint Problem for Steady State Heat Conduction id=sec:adjoint
+
+In this section, we are interested in solving the following PDE-constrained optimization problem from [eq:optimization] for steady state heat conduction:
+\begin{equation} \label{eqn:optimization_problem}
+	\begin{aligned}
+	& \min_{\mathbf{p}}
+	& & f\left(T,\mathbf{p}\right) = \frac{1}{2} \sum_{i=1}^{N} \left( T_i - \bar{T}_i \right)^2 , \\
+	& \text{subject to}
+	& & \mathcal{R}\left(T,\mathbf{p}\right)=\nabla \cdot \kappa \nabla T + g_b =0, & \text{in}~\Omega , \\
+\end{aligned}
+\end{equation}
+where $f$ is the objective function from [eq:objective] without regularization, $g_b$ is the distributed heat flux, $T$ is the experimental temperature field being compared to our simulation temperature at discrete locations, $\bar{T}_i$.  Other forms for the objective function are possible such as different norms or different types of measurements that may require integration over a line or volume.
```

Comment:
isnt it flipped? T_bar is experimental, T is the variable

Reply:
fixed